lib/web_ui/test/engine/semantics/semantics_test.dart

```diff
@@ -190,8 +191,30 @@ void _testEngineSemanticsOwner() {
       semantics().semanticsEnabled = false;
     });
   });
+  test('checks shouldEnableSemantics for every global event', () {
+    final MockSemanticsEnabler mockEnableSemantics = MockSemanticsEnabler();
+    semantics().enableSemantics.enableSemantics = mockEnableSemantics;
```

Comment:
`enableSemantics.enableSemantics` doesn't look right. Let's rename the field (and the setter) to `semanticsEnabler`.

Reply:
Done.

I changed the names. I will do the refactor we discussed in a later PR.